src/app/beer_garden/garden.py

```diff
@@ -57,6 +57,22 @@ def local_garden() -> Garden:
     )
 
 
+def sync_garden():
+
+    publish(
+        Event(
+            name=Events.GARDEN_UPDATED.name,
```

Comment:
Let's create a new event for this - I think it will be better for flexibility if we can treat updates that happen "organically" differently from sync operations.

Reply:
Done